ports/stm32/Makefile

```diff
@@ -578,7 +578,7 @@ define RUN_DFU
 	$(ECHO) "Writing $(1) to the board"
 	$(if $(filter $(USE_PYDFU),1),\
 	$(Q)$(PYTHON) $(PYDFU) --vid $(BOOTLOADER_DFU_USB_VID) --pid $(BOOTLOADER_DFU_USB_PID) -u $(1),
-	$(Q)$(DFU_UTIL) -a 0 -d $(BOOTLOADER_DFU_USB_VID):$(BOOTLOADER_DFU_USB_PID) -D $(1))
+	$(Q)$(DFU_UTIL) -a 0 -d $(BOOTLOADER_DFU_USB_VID):$(BOOTLOADER_DFU_USB_PID) -D $(1) -a 0 -s :leave)
```

Comment:
is the second `-a 0` redundant?

Reply:
No it just adds a second command to set the leave modifier which can't be se with `-d`

Reply:
Sorry, right the secon `-a 0` is redundant, although it's harmless. The `-s` is to allow setting modifiers (without an address). Will remove `-a 0`, thanks!

Reply:
Originally I had this working with 2 `dfu-util` commands.

Reply:
Fixed